components/script/dom/bindings/conversions.rs

```diff
@@ -62,6 +62,7 @@ use servo_config::opts;
 use std::{char, ffi, ptr, slice};
 
 /// A trait to check whether a given `JSObject` implements an IDL interface.
+#[must_root]
```

Comment:
Why is this change required?

Reply:
Like mentioned somewhere before - some heuristics. It is a trait that is implemented by `#[must_root]` types, I believe.